parser/src/string.rs

```diff
@@ -0,0 +1,155 @@
+use crate::ast::{Constant, Expr, ExprKind, Location};
+use crate::error::LexicalErrorType::FStringError;
+use crate::error::{LexicalError, LexicalErrorType};
+use crate::fstring::parse_located_fstring;
+use crate::token::StringKind;
+use itertools::Itertools;
+
+pub fn parse_implicit_concatenation(
```

Comment:
The function is not only handling concatenation but strings itself now
```suggestion
pub fn parse_strings(
```